samples/attested_tls/tls_between_enclaves/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+cmake_minimum_required(VERSION 3.11)
+
+project("Sample: TLS connection between two enclaves" LANGUAGES C CXX)
+
+set(CMAKE_CXX_STANDARD 11)
+
+add_custom_target(tls_between_enclaves
```

Comment:
If this is a new project, how does it know about these targets? I don't see any subdirectories getting included.

Reply:
Those two directories (tls_between_enclaves and tls_between_non_enclave_enclave) exist for the purpose of helping sample users to know what's needed to for establishing its corresponding tls channel type